### ISSUE TITLE
What requirements should we have reporting on/mid/off-peak electricity use?

### ISSUE THREAD

**Issue by macintoshpie** [BuildingSync/TestSuite#35]
STD 211, 6.2.2.1.a.8 requires "Actual or billed (or both if reported) peak electric demand (kW), identified as on-peak, midpeak, and off-peak" for Historical Monthly Utility Data.

First task is to determine what this actually means, and we have two hypotheses:
1. report 3 peaks identified for a 'period' (ie auc:TimeSeries), one for the on-peak, mid-peak, off-peak
2. report 1 peak identified for a 'period' (ie auc:TimeSeries), and classify the peak as having occurred on-peak, mid-peak, or off-peak

Second task is to determine whether this is only applicable to Utilities whose rate schedule can have on-peak, mid-peak, or off-peak intervals.

Here's an example image to help think through this. The colored bars represent cost for a billing period, the line represents electricity use. The circles represent global maxima for the period (in this case a day) for each billing tier.
![image (1)](https://user-images.githubusercontent.com/18518728/93511741-840c9000-f8e0-11ea-852b-342d774822c3.png)

Currently we require for all auc:ResourceUses that have an energy type of Electricity to report a peak value for each month, for example:
```xml
<auc:TimeSeries ID="Scenario-A-TS13">
  <auc:ReadingType>Peak</auc:ReadingType>
  <auc:StartTimestamp>2019-01-01T00:00:00</auc:StartTimestamp>
  <auc:EndTimestamp>2019-02-01T00:00:00</auc:EndTimestamp>
  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
  <auc:IntervalReading>14.78</auc:IntervalReading>
  <auc:ResourceUseID IDref="ResourceUse-Electricity" />
</auc:TimeSeries>
```

**Comment by macintoshpie** [comment]
Decided to add a new element indicating the "type" of Peak reading, e.g. `auc:PeakType`, which includes the enums:
- On-peak
- Mid-peak
- Off-peak
- Unknown

**Comment by macintoshpie** [comment]
Addressed in #37
